python-test.py

```diff
@@ -1,11 +1,11 @@
 print ("This is a tttsting code to check error catching")
 
-def simple_fn(range):
-    if range > 0:
+def simple_fn(range, seed):
+    if range >= 0 and seed == 1:
```

Comment:
Consider renaming the parameter `range` to avoid conflicting with the built-in function